spec/app2_spec.rb

```diff
@@ -154,10 +154,26 @@
     )
   end
 
+  it do
+    expect(chef_run).to disable_osl_app('replicant-redmine-unicorn').with(
+      user: 'replicant',
```

Comment:
For this one, don't worry about all the parameters since you already tested that above.